yb-voyager/cmd/analyzeSchema.go

```diff
@@ -461,14 +461,19 @@ func checkDDL(sqlInfoArr []sqlInfo, fpath string) {
 			partitionColumnsList := utils.CsvStringToSlice(partitionColumns)
 			primaryKeyColumnsList := utils.CsvStringToSlice(primaryKeyColumns)
 			sort.Strings(primaryKeyColumnsList)
+			if len(primaryKeyColumnsList) == 0 || len(allColumnsList) == 1 {
+				continue
+			}
+			if len(allColumnsList) > 1 && !strings.Contains(strings.ToLower(allColumnsList[len(allColumnsList)-2]), "primary key") {
+				continue
+			}
```

Comment:
Please add comments explaining the meaning of these statements.

Reply:
+1 to this comment, mention somewhere in a comment how the regex would be broken up and which submatches correspond to which parts